Tabloid/client/src/components/AddCategory.js

```diff
@@ -12,14 +12,25 @@ export const AddCategoryForm = () => {
 
   const submit = (evt) => {
     evt.preventDefault();
-    addCategory(category).then(() => history.push("/category"));
+
+    const input = document.querySelector("#newCatInp").value;
+    if (
+      !input ||
```

Comment:
I think you can shorten this to `!input || !input.trim()`

Reply:
shortened! @joshdbarton